packages/fondue/src/components/RichTextEditor/components/Toolbar/Tooltip.tsx

```diff
@@ -0,0 +1,45 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React from 'react';
+import * as TooltipPrimitive from '@radix-ui/react-tooltip';
+
+export const TooltipProvider = TooltipPrimitive.Provider;
+export const Tooltip = TooltipPrimitive.Root;
+export const TooltipTrigger = TooltipPrimitive.Trigger;
+export const TooltipPortal = TooltipPrimitive.Portal;
+export const TooltipContent = TooltipPrimitive.Content;
+
+export function withTooltip<T extends React.ComponentType<any> | keyof HTMLElementTagNameMap>(Component: T) {
+    // eslint-disable-next-line react/display-name
```

Comment:
Cant you just add a displayName instead of disabling it?

Reply:
Added displayname everywhere where it was needed